packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -139,8 +139,7 @@ class InspectorController extends DisposableController
     debugSummaryLayoutEnabled.value = !debugSummaryLayoutEnabled.value;
   }
 
-  // TODO(albertusangga): Remove this flag if required CL to Flutter is landed
-  static bool enableExperimentalStoryOfLayout = false;
+  static bool enableExperimentalStoryOfLayout = true;
```

Comment:
So sorry for this one, I should have removed it completely, this is unnecessary to enable the layout explorer